### PR TITLE
Adding Logging

### PR DESCRIPTION
**Pull request by Frstrm**
- config options
- log method

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -7,5 +7,10 @@
   "stealthy-ignoreFriendlyStealth-name": "Ignore friendly stealth",
   "stealthy-ignoreFriendlyStealth-hint": "Perception vs Stealth tests will be ignored for tokens of the same disposition",
   "stealthy-ignoreFriendlyUmbralSight-name": "Ignore friendly Umbral Sight",
-  "stealthy-ignoreFriendlyUmbralSight-hint": "In darkvision mode, ignore the Umbral Sight feature for tokens of the same disposition"
+  "stealthy-ignoreFriendlyUmbralSight-hint": "In darkvision mode, ignore the Umbral Sight feature for tokens of the same disposition",
+  "stealthy-debugLogging-name": "debugLoggin",
```
Comment (Eligarf) [inline lang/en.json:11]: need a 'g' at end

File: module.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
       "name": "Frstrm"
     }
   ],
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (Eligarf) [inline module.json:13]: lol, which ever of us ends up second will have to adjust this number

**Review by Eligarf — CHANGES_REQUESTED** [review]

File: module.json (hunk 2 above)
Comment (Eligarf) [reply module.json:13]: I'll bump, never mind

**Comment by Eligarf** [comment]
Also need entry in ChangeLog.md for this

File: ChangeLog.md [hunk 3]
```diff
@@ -6,6 +6,10 @@
 * Github workflows support
 * Use ATL 50% token alpha as fallback if no TokenMagicFX present
 
+
+### Version 1.2.1
```
Comment (Eligarf) [inline ChangeLog.md:10]: Gah, yours was supposed to go in first.... Sigh. go ahead and move this into the right place

File: module.json (hunk 2 above)
Comment (Eligarf) [inline module.json:13]: looks like 1.2.1 is ghosted. So, new work needs to be based on 'dev' not 'main' to prevent this foolery in the future.

**Review by Eligarf — CHANGES_REQUESTED** [review]

**Comment by Eligarf** [comment]
You don't actually use the logging anywhere...

File: module.json (hunk 2 above)
Comment (Frstrm) [reply module.json]: bumping to 1.2.3

File: ChangeLog.md (hunk 3 above)
Comment (Frstrm) [reply ChangeLog.md]: bumping to 1.2.3

File: ChangeLog.md [hunk 4]
```diff
@@ -10,7 +10,7 @@
 * Use ATL 50% token alpha as fallback if no TokenMagicFX present
 
 
-### Version 1.2.1
+### Version 1.2.3
```
Comment (Eligarf) [inline ChangeLog.md:13]: Move this up above 1.2.2

**Review by Eligarf — CHANGES_REQUESTED** [review]

**Review by Eligarf — APPROVED** [review]

**Pull request merged**